src/components/error/FrontEnd/FrontEndError.react.js

```diff
@@ -40,9 +41,12 @@ class FrontEndError extends Component {
                     </span>
                 </span>
                 <span className="dash-fe-error-top__group">
-                    <span className="dash-fe-error__timestamp">
+                    <span className="dash-fe-error__timestamp percy-hide">
                         {`${e.timestamp.toLocaleTimeString()}`}
                     </span>
+                    <span className="dash-fe-error__timestamp percy-show">
+                        {'12:49:14 PM'}
+                    </span>
```

Comment:
😸 
![image](https://user-images.githubusercontent.com/1280389/56146320-04a0c180-5f74-11e9-9298-c541feb42d1e.png)

Reply:
Hide the time-varying timestamps on percy and display a constant timestamp so that we can still see what the UI looks like.